From 399913fa0da6dbda9c4a29a46996f7f44a58e7cf Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Mon, 16 Mar 2020 05:55:14 +0200
Subject: [PATCH] [SW-0] goya2: update get_cs_pad_sel()

It is connected to QSPI block and not SPI block
So use the correct PAD number

Change-Id: I0d422de6a94447a09e0a29bfd3c98cb0737aca44
---

diff --git a/goya2/common.c b/goya2/common.c
index 233c339..7a097b5 100644
--- a/goya2/common.c
+++ b/goya2/common.c
@@ -174,7 +174,7 @@
 
 u32 get_cs_pad_sel(void)
 {
-	return mmPSOC_GLOBAL_CONF_PAD_SEL_53;
+	return mmPSOC_GLOBAL_CONF_PAD_SEL_19;
 }
 
 u32 get_cpu_boot_status_reg(void)
diff --git a/goya2_arc/common.c b/goya2_arc/common.c
index c422415..5a670d7 100644
--- a/goya2_arc/common.c
+++ b/goya2_arc/common.c
@@ -165,7 +165,7 @@
 
 u32 get_cs_pad_sel(void)
 {
-	return mmPSOC_GLOBAL_CONF_PAD_SEL_53;
+	return mmPSOC_GLOBAL_CONF_PAD_SEL_19;
 }
 
 u32 get_cpu_boot_status_reg(void)
